packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -570,6 +584,18 @@ export class Function extends FunctionBase {
     }
 
     this.currentVersionOptions = props.currentVersionOptions;
+
+    if (props.filesystems) {
+      // add dependency when necessary
+      // for (const fs of props.filesystems) {
+      //   this.node.addDependency(fs.config.dependency);
+      // }
+      resource.addPropertyOverride('FileSystemConfigs',
```

Comment:
I would move this before the `new CfnFunction()` call so you don't need to use an override.

Reply:
@jogold I didn't get it. Can you share some code snippet?

Reply:
Just saw that the L1 has not been updated yet...